plugins/pgpointcloud/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ PDAL_ADD_PLUGIN(reader_libname reader pgpointcloud
         ${LIBXML2_LIBRARIES}
     INCLUDES
         ${LIBXML2_INCLUDE_DIR}
+        ${NLOHMANN_INCLUDE_DIR}
```

Comment:
this shouldn't be needed in pgpointcloud

Reply:
These were both in the list of things that were erroring on compile when I added nlohmann to QuickInfo.hpp